components/GameForm.tsx

```diff
@@ -32,6 +32,9 @@ interface GameFormProps {
 function GameForm(props: GameFormProps) {
   const { title, gameId, onSubmit } = props;
 
+  const [game, setGame] = useState<any>([]);
```

Comment:
you exactly know what should be there, please avoid using `any` if you really do not have to